Makefile

```diff
@@ -1,7 +1,51 @@
+######################
 # Define a variable for the Terraform examples directory
-TERRAFORM_DIR := examples/rosa-classic-with-unmanaged-oidc
+# TERRAFORM_DIR := examples/rosa-classic-public
+TERRAFORM_DIR := examples/rosa-classic-public-with-byo-vpc
+# TERRAFORM_DIR := examples/rosa-classic-public-with-byo-vpc-byo-iam-byo-oidc
+# TERRAFORM_DIR := examples/rosa-classic-public-with-idp-machine-pools
+# TERRAFORM_DIR := examples/rosa-classic-public-with-unmanaged-oidc
 
-verify:
+######################
+# Log into your AWS account before running this make file.
+# Create .env file with your ROSA token. This file will be ignored by git.
+# format.
+# RHCS_TOKEN=<ROSA TOKEN>
+
+include .env
+export $(shell sed '/^\#/d; s/=.*//' .env)
+TF_VAR_RHCS_TOKEN=$RHCS_TOKEN
+TF_VAR_AWS_DEFAULT_REGION := $(shell aws configure get region)
```

Comment:
I believe those lines are not needed as the providers use the origin environment variables (and not the variable), no?